dataflow/gemma/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This uses Ubuntu with Python 3.11
+ARG SERVING_BUILD_IMAGE=tensorflow/tensorflow:latest-gpu
+
+FROM ${SERVING_BUILD_IMAGE}
+
+WORKDIR /workspace
+
+RUN apt-get update -y
+RUN apt-get update
+RUN apt-get install cmake -y
+
+COPY requirements.txt requirements.txt
+RUN pip install --upgrade pip \
+    && pip install --no-cache-dir -r requirements.txt \
+    && rm -f requirements.txt
```

Comment:
nit: note that removing requirements will not free space on the image in this case, so i probably wouldn't worry about it.

Reply:
removed